src/core/objects/object_mover.cpp

```diff
@@ -124,18 +131,31 @@ void ObjectMover::move(qint32 dx, qint32 dy, bool move_opposite_handles)
 	}
 	
 	// Apply handle constraints
-	if (move_opposite_handles)
+	for (auto& constraint : handle_constraints)
 	{
-		for (auto& constraint : handle_constraints)
+		MapCoord anchor_point = constraint.object->getCoordinate(constraint.curve_anchor_index);
+		MapCoord moved_handle = constraint.object->getCoordinate(constraint.moved_handle_index);
+		MapCoordF to_hover_point = MapCoordF(moved_handle - anchor_point);
+		MapCoord control = constraint.object->getCoordinate(constraint.opposite_handle_index);
```

Comment:
Now looking at this again, I wonder if we can't use a better name than `control` here, e.g. `opposite_handle`.

Reply:
Well, from my point of view the rename is out of scope of this change. The task is to implement corner click-in editing mode and this is the original code that gets rearranged towards the desired goal. By default I don't change the original code more than necessary. That said, I don't have strong opinion about the naming and I'm fine with either leaving it as is, which is expressed by this patch set, or renaming it to obviously more logical `opposite_handle`.

Reply:
I know that the name comes from the original code. I just can't skip taking such a note when the unit is touched anyway. It can go to a separate commit, but I wouldn't want to enforce a separate PR, with all the CI cycles.

Reply:
Fine with me and I don't object incremental refactoring. IMO all these changes can easily go into the last patch, now labeled "Use in-class member initializers ".

Talking about these exact numbers, why stop at the rename? Mapper Coding conventions encourage use of AAA. Guidelines add ES.23 and Con.4. Opinions?
```suggestion
		const auto anchor_point = constraint.object->getCoordinate(constraint.curve_anchor_index);
		const auto moved_handle = constraint.object->getCoordinate(constraint.moved_handle_index);
		auto opposite_handle = constraint.object->getCoordinate(constraint.opposite_handle_index);
		auto to_hover_point = MapCoordF { moved_handle - anchor_point };
```

Reply:
Accepting the suggestion would break the code as it renames only the variable declaration but not the subsequent use. :)
Thanks for clarification on the expected extent of refactoring. For now I've just made the rename as part of the feature implementation.